test/wallet-test.js

```diff
@@ -1642,4 +1643,298 @@ describe('Wallet', function() {
       }
     });
   });
+
+  describe('TXDB locked balance', function() {
+    const network = Network.get('regtest');
+    const workers = new WorkerPool({ enabled });
+    const wdb = new WalletDB({ network, workers });
+    const name = 'satoshi';
+    const value = 1e6;
+    const lockup = 2e6;
+    const fee = 10000;
+    let wallet;
+
+    before(async () => {
+      await wdb.open();
+      wallet = await wdb.create();
+      // rollout all names
+      wdb.height = 52 * 144 * 7;
+    });
+
+    after(async () => {
+      await wdb.close();
+    });
+
+    it('Fund wallet', async () => {
```

Comment:
Most tests in `test/*.js` seems to follow a convention of starting with the word "should", maybe do something like this for all test cases in this PR?
```js
it('should fund wallet', async () => { ... })
```

Reply:
Thanks! Truth is, I'm not a huge fan of the "should" convention and I'm always trying to break it, especially in a case like this where the entire `Describe` block is testing one thing, and the `it` blocks are just like incremental steps :-)

But, I'm so happy you reviewed my PR I'll take your suggestion!